package.json

```diff
@@ -44,13 +44,13 @@
     "ethereumjs-util": "^6.0.0",
     "ethers": "^4.0.13",
     "ethjs-abi": "^0.2.1",
-    "ganache-cli": "=6.1.8",
+    "ganache-cli": "=6.2.5",
     "solhint": "^1.2.1",
-    "truffle": "^4.1.14"
+    "truffle": "^5.0.0"
```

Comment:
no need to run `npm install -g truffle`?

Reply:
to use bundled truffle: `./node_modules/.bin/truffle ...` ← this will be a truffle version from package.json